accumulator/utils.go

```diff
@@ -4,28 +4,60 @@ import (
 	"fmt"
 	"math/bits"
 	"sort"
+	"sync"
 )
 
 // verbose is a global const to get lots of printfs for debugging
 var verbose = false
 
+// PositionList is a wrapper around slice of ints.
+type PositionList struct {
```

Comment:
nit: Could this not just be:
```go
type PositionList []uint64
```
to avoid code like: `positionList.list`

Reply:
I'll modify it and push it along with better getRootsReverse (and make getRootsForwards a separate function) as talked about in the utreexo meeting call.

Reply:
Ok so change of mind on the `type PositionList []uint64` because then the function would have to ask for `*PositionList` and not `*[]uint64`.

You could get around that problem by type casting but then you have to do this shimmy
```
rows := treeRows(numLeaves)
positionList := NewPositionList()
defer positionList.Free()

x := []uint64(*positionList)
ProofPositions(targets, numLeaves, rows, &x)
```

which arguably is worse than positionList.list